### PR TITLE
[issue-38] Formatted README.md

### PR DESCRIPTION
**Pull request by Aparnarr**
Signed-off-by: Aparna <Aparna.Rr@emc.com>

Change log Description:
 1. Fixed formatting issues
 2. Edited content(minor changes)

Purpose of the change:
  Fixes #38, #41

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -157,17 +160,16 @@ spec:
           claimName: pravega-tier2
 ```
 
-where:
```
Comment (adrianmo) [inline README.md]: This is not a note. It's a description of the variables in the example snippet. So I should leave it as it is.

File: README.md [hunk 2]
```diff
@@ -268,17 +270,15 @@ docker tag pravega/pravega-operator [REGISTRY_HOST]:[REGISTRY_PORT]/pravega/prav
 docker push [REGISTRY_HOST]:[REGISTRY_PORT]/pravega/pravega-operator
 ```
 
-where:
```
Comment (adrianmo) [inline README.md]: Not a note. Please revert change.

File: README.md [hunk 3]
```diff
@@ -1,51 +1,50 @@
 # Pravega Operator
 
-### Project status: in development
```
[the remark this answers is not in the document]
Comment (Aparnarr) [reply README.md]: sure! will edit the doc based on ur review

File: README.md (hunk 2 above)
Comment (Aparnarr) [reply README.md]: @adrianmo 
i will add the below mentioned content and create TOC 👍

**Comment by Aparnarr** [comment]
This PR is closed and replaced with PR #57

**Comment by Aparnarr** [comment]
This PR is closed and replaced with PR #57

**Pull request closed without merging**